charts/newrelic-infrastructure/values-windows-linux.yaml

```diff
@@ -0,0 +1,14 @@
+global:
```

Comment:
i couldn't find anywhere that uses this value in this PR. is this needed?

Reply:
Yes, global values are required. The New Relic common library uses these values and set them globally (like LicenseKey)[Reference]( https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/README.md#affinities-and-tolerations )
This is an example yaml file added for windows integration.